storage/google/cloud/storage/blob.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# pylint: disable=too-many-lines
```

Comment:
Should we make an issue/reminder to break this file up down the road?

I can make the issue if you think that's an ok plan.